tests/models/test_amp.py

```diff
@@ -173,3 +174,38 @@ def test_cpu_model_with_amp(tmpdir):
 
     with pytest.raises((MisconfigurationException, ModuleNotFoundError)):
         tpipes.run_model_test(trainer_options, model, on_gpu=False)
+
+
+def test_amp_without_apex(tmpdir):
```

Comment:
i don't understand what this is testing

Reply:
this is testing that even with default amp type without requesting percsion=16 the amp backend is void...